shell/probe/nocturnal.sh

```diff
@@ -16,11 +18,9 @@ do
         echo "Downloading $uuid"
         curl -sf --max-redirs 0 --create-dirs -o "$vst/$uuid" $task
         chmod +x "$vst/$uuid"
-        CDIR="${PWD}"
-        cd "$vst" &>/dev/null
         echo "Invoking $uuid"
-        $vst/$uuid & test_pid=$!
-        cd "$CDIR" &>/dev/null
+        cd "$vst" || exit
```

Comment:
maybe log and/or exit with non-0 code?

Reply:
I went with a non-zero exit code, because you would get an indication of what failed already. I went with 1, because it's a pretty normal "ehhhh, this failed" error code. 

```
:) cat y
vst="somedir"

cd "$vst" || echo "cd to $vst failed"; exit

:) bash y
y: line 3: cd: somedir: No such file or directory
cd to somedir failed

:) dash y
y: 3: cd: can't cd to somedir
cd to somedir failed
```